affected_purls:
- pkg:pypi/django@2.2
- pkg:pypi/django@2.2.1
- pkg:pypi/django@2.2.10
- pkg:pypi/django@2.2.11
- pkg:pypi/django@2.2.12
- pkg:pypi/django@2.2.13
- pkg:pypi/django@2.2.14
- pkg:pypi/django@2.2.15
- pkg:pypi/django@2.2.16
- pkg:pypi/django@2.2.17
- pkg:pypi/django@2.2.18
- pkg:pypi/django@2.2.19
- pkg:pypi/django@2.2.2
- pkg:pypi/django@2.2.20
- pkg:pypi/django@2.2.21
- pkg:pypi/django@2.2.22
- pkg:pypi/django@2.2.23
- pkg:pypi/django@2.2.24
- pkg:pypi/django@2.2.25
- pkg:pypi/django@2.2.26
- pkg:pypi/django@2.2.3
- pkg:pypi/django@2.2.4
- pkg:pypi/django@2.2.5
- pkg:pypi/django@2.2.6
- pkg:pypi/django@2.2.7
- pkg:pypi/django@2.2.8
- pkg:pypi/django@2.2.9
- pkg:pypi/django@3.2
- pkg:pypi/django@3.2.1
- pkg:pypi/django@3.2.10
- pkg:pypi/django@3.2.11
- pkg:pypi/django@3.2.2
- pkg:pypi/django@3.2.3
- pkg:pypi/django@3.2.4
- pkg:pypi/django@3.2.5
- pkg:pypi/django@3.2.6
- pkg:pypi/django@3.2.7
- pkg:pypi/django@3.2.8
- pkg:pypi/django@3.2.9
- pkg:pypi/django@4.0
- pkg:pypi/django@4.0.1
aliases:
- CVE-2022-23833
- GHSA-6cw3-g6wv-c2xv
fixed_purl:
- pkg:pypi/django@2.2.27
- pkg:pypi/django@3.2.12
- pkg:pypi/django@4.0.2
references:
- id: 164962
  reference_id: ''
  url: https://docs.djangoproject.com/en/4.0/releases/security/
- id: 160622
  reference_id: ''
  url: https://groups.google.com/forum/#!forum/django-announce
- id: 160627
  reference_id: ''
  url: https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/B4SQG2EAF4WCI2SLRL6XRDJ3RPK3ZRDV/
- id: 166930
  reference_id: ''
  url: https://security.netapp.com/advisory/ntap-20220221-0003/
- id: 166931
  reference_id: ''
  url: https://www.debian.org/security/2022/dsa-5254
- id: 166929
  reference_id: ''
  url: https://www.djangoproject.com/weblog/2022/feb/01/security-releases/
- id: 167671
  reference_id: CVE-2022-23833
  url: https://nvd.nist.gov/vuln/detail/CVE-2022-23833
- id: 167672
  reference_id: GHSA-6cw3-g6wv-c2xv
  url: https://github.com/advisories/GHSA-6cw3-g6wv-c2xv
severities:
- id: 25535
  reference_id: 167672
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: HIGH
summary: Infinite Loop in Django
vulnerability_id: VCID-b6ga-9kg7-aaam
weaknesses: []
